website/docs/requests.mdx

```diff
@@ -41,7 +41,7 @@ The following plans include dedicated API and CDN nodes.
 
 #### Hosted
 
-Runs on dedicated servers provided by ConfigCat.
+Runs on dedicated infrastructure provided by ConfigCat.
 The basic package includes:
```

Comment:
Is this infrastructure setup still relevant? Haven't we migrated the hosted plans to AWS? Or will we very soon?

Reply:
We will migrate them to AWS, but this plan and setup is still relevant.